rest/router.go

```diff
@@ -45,6 +45,7 @@ func GetRouter(pool *pgxpool.Pool, defaultApiAddresses []string) *gin.Engine {
 		nft.GET("/creator", handleNftCreators)
 		nft.GET("/user-stat", handleNftUserStat)
 		nft.GET("/marketplace", handleNftMarketplaceItem)
+		nft.GET("/collector-in-creators-top-k", handleNftCollectorInCreatorsTopKRequest)
```

Comment:
brainstorming better names...
```suggestion
		nft.GET("/collector-top-ranked-creators", handleNftCollectorInCreatorsTopKRequest)
```